src/components/ebay-breadcrumbs/index.marko

```diff
@@ -10,6 +10,7 @@ static var ignoredAttributes = [
 ];
 
 $ input.toJSON = noop;
+$ var anyHref = input && input.items && input.items.some(element => element.href != null);
```

Comment:
Sorry, just noticed this. No need to check for `input`.
You can do something like: 
`(input.items || []).some(...etc)`